README.md

```diff
@@ -271,11 +276,12 @@ $layout-direction: ltr;
 @import 'layouts';
 @import 'views';
 @import 'overrides';
-
 ```
 
 **NOTE**: Since both libSass and RubySass lack a way to pass in variables/configuration into their ``@import {file}`` method, each app is responsible for 1) storing any npm-based dependencies, including the edx-pattern-library, in the best directory structure for that app's set up and 2) creating a ``_lib.scss`` partial to import all third party library dependencies from that structure for the Pattern Library to use (see above example).
 
+**NOTE**: We support right-to-left and left-to-right-based layouts. To suppor this within each app, we recommend including a ``_rtl.scss`` or ``_ltr.scss`` file that contains the right layout setting and reference to specific third party utilities.
```

Comment:
When you say "we recommend", does this mean that you can choose not to include one or the other? Does it matter that ``$layout-direction`` won't be set?